cropharvest/eo/eo.py

```diff
@@ -38,7 +38,9 @@
 
 
 class EarthEngineExporter:
-    def __init__(self, data_folder: Path = DATAFOLDER_PATH) -> None:
+    def __init__(
+        self, data_folder: Path = DATAFOLDER_PATH, labels: Optional[geopandas.GeoDataFrame] = None
+    ) -> None:
```

Comment:
A refactor that made the whole data pipeline in `crop-mask` much cleaner and easier to work with was separating **label management** (reading in, filtering, creating folders) and **label actions** (exporting, engineering). 

My fairly new opinion is that the Exporter class should be for exporting only, it doesn't need to know about where and how the data is read in, stored or filtered (similarly Engineer doesn't need to know about this either). Having both a `data_folder` parameter and a `labels` parameter is kind of a middle ground between this, I would suggest eliminating the data_folder path completely. What do you think? This is more work so we can defer it to later too though

Reply:
yes, this makes sense to me

Reply:
But defer to later?

Reply:
yes - i think we could do away with the concept of a data folder elsewhere in the codebase too